app/controllers/cards_controller.rb

```diff
@@ -35,6 +34,21 @@ def destroy
     redirect_to cards_path
   end
 
+  def check
+    find_text = params[:card][:check_original_text].strip
+    id_card = params[:card][:id]
+    @card = Card.find(id_card)
+    if find_text.empty?
+      flash[:error] = 'Пустое значение! Попробуйте еще раз...'
+    elsif @card.original_text.include?(find_text)
```

Comment:
If i will check with card original text is 'FooBar'
and sent 'F' as answer, it will pass.